tests/Makefile.am

```diff
@@ -0,0 +1,89 @@
+# Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
I don't really know how this autotest stuff works, and I am trying to avoid learning if possible.
It would be nice if the extensions/ folder was used for testing as well - so you can actually "merge" the tests here with whatever tests/ folders may exist in any extension.